docs/dorgeclipseopenj9criuimmutableenvvars.md

```diff
@@ -0,0 +1,57 @@
+<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -Dorg.eclipse.openj9.criu.ImmutableEnvVars
+
+This system property specifies the environment variables that the VM can detect on restore.
```

Comment:
> VM can detect on restore.

Not quite. During the checkpoint phase, the VM detects env vars in the ImmutableEnvVars list. During the restore phase the VM detects env vars in the ImmutableEnvVars list + any env vars added via `CRIUSupport.registerRestoreEnvVariables`